generators/network/templates/start.sh

```diff
@@ -6,7 +6,17 @@
 #
 # Exit on first error, print all commands.
 set -ev
-for CONTAINER in $(docker ps -f label=fabric-environment-name="<%= name %>" -q -a); do
-    docker start ${CONTAINER}
-done
+CONTAINER=$(docker ps -f label=fabric-environment-name="<%= name %>" -q -a)
+if [ -z "$CONTAINER" ]
+then
+    export MICROFAB_CONFIG='<%-microfabConfig%>'
+    docker run -e MICROFAB_CONFIG --label fabric-environment-name="<%= name %>" -d -p <%-port%>:<%-port%> ibmcom/ibp-microfab:0.0.4
+else
+    for CONTAINER in $(docker ps -f label=fabric-environment-name="<%= name %>" -q -a); do
```

Comment:
Isn't there only one container? Thought iterating like this will make sure we don't try to start a container that doesn't exist without explicit code for it.

Reply:
True - I can remove this loop and change it to just start the container! (As we know the container ID already!)

Reply:
Sounds good!